projects/components/src/textarea/textarea.component.test.ts

```diff
@@ -30,7 +31,7 @@ describe('Textarea Component', () => {
       }
     });
 
-    expect(logSpy).not.toHaveBeenCalled();
+    expect(spectator.inject(LoggerService).warn).toHaveBeenCalled();
```

Comment:
this should be a negated check, right?

Reply:
The test should've failed 🙃

Reply:
Yeah, is it even working?

Reply:
Updated the test. It worked because the `Loggerservice` was mocked on the global level, warn was called once.